### PR TITLE
DOCS-722: Lazy load video gifs

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: layouts/partials/scripts.html [hunk 1]
```diff
@@ -73,6 +73,7 @@
 <!-- scripts for prism -->
 <script src='{{ "js/prism.js" | relURL }}'></script>
 {{ end }}
+<script type="module" src='{{ "js/yall.js" | relURL }}'></script>
```
Comment (mrloureed) [inline layouts/partials/scripts.html:76]: Is this a lazy loading library. You may be able to write a script that uses intersection observer instead.

Comment (npentrel) [reply layouts/partials/scripts.html:76]: yes - using https://github.com/malchata/yall.js . Does it look like I am importing that correctly? I could probably write my own logic... yes... but why reinvent the wheel?

File: layouts/docs/content.html [hunk 2]
```diff
@@ -26,7 +26,7 @@ <h4 class="alert-heading">DRAFT</h4>
 			<div class="col hover-card hover-card-small">
 			  <a href="{{.RelPermalink}}">
 				{{ if .Params.webmSrc }}
-				  <video autoplay loop muted playsinline preload="auto" alt="{{ .Params.videoAlt }}" width="100%">
+				  <video autoplay loop muted playsinline alt="{{ .Params.videoAlt }}" width="100%" class="lazy">
```
[the remark this answers is not in the document]
Comment (npentrel) [reply layouts/docs/content.html:29]: these are always gif-like videos

**Comment by npentrel** [comment]
Hey @mrloureed I got this to work with [lozad.js](https://github.com/ApoorvSaxena/lozad.js/). I've decided I'd prefer using a library over creating my own logic with intersection observers - since I won't be able to do it better than these and this is super lightweight. If you can give this a quick review when you're back to check that it's working and that I'm including this as it should be, I'd appreciate that :).

**Review by mrloureed — APPROVED** [review]
LGTM!

**Pull request merged**